backend/drm/drm.c

```diff
@@ -720,8 +720,7 @@ static bool drm_connector_set_cursor(struct wlr_output *output,
 	}
 
 	struct gbm_bo *bo = plane->cursor_enabled ? plane->surf.back : NULL;
-
-	if (drm->parent) {
+	if (bo && drm->parent) {
 		bo = copy_drm_surface_mgpu(&plane->mgpu_surf, plane->surf.back);
```

Comment:
We should also replace `plane->surf.back` by `bo` here, just to prevent future mistakes in case `plane->surf.back`is replaced by something else.

Reply:
Oh yeah, that's a great idea